app/javascript/elm/src/Main.elm

```diff
@@ -670,7 +686,7 @@ viewMobileFilters model =
         [ viewParameterFilter model.sensors model.selectedSensorId
         , viewSensorFilter model.sensors model.selectedSensorId
         , viewLocation model.location model.isIndoor
-        , TimeRange.view RefreshTimeRange
+        , TimeRange.view RefreshTimeRange False
```

Comment:
I wonder if creating a custom type instead of the boolean would simplify or not the code. Something like
```
type Streaming
  = Active
  | Dormant
```
so that here you could use
```
 TimeRange.view RefreshTimeRange Dormant
```

Reply:
I'll consider changing that, for sure it would be more meaningful, I didn't really like passing the boolean like that because we don't know what it represents until we look for it